src/components/ContactMe/ContactInfoCard/styles.css

```diff
@@ -0,0 +1,33 @@
+.contact-details-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background: #130f2a;
+  border-radius: 9px;
+  border: 1.5px solid #6751b9;
+  padding: 24px;
+  margin-bottom: 32px;
+}
+
+.contact-details-card .icon {
+  width: 64px;
+  height: 64px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: 8px;
+  background: #3d3072;
+  margin-bottom: 14px;
+}
+
+.icon img {
+  width: 30px;
+  height: auto;
+  object-fit: contain;
+}
+
+.contact-details-card p {
+  font-size: 15px;
+  font-weight: 400;
+}
```

Comment:
remove this style. using Text component will automatically set all necessary styles.